### DISCUSSION TITLE
Can multiple versions of brew coexist on a system?

### DISCUSSION THREAD

**Discussion by thomasaarholt** [Homebrew/discussions#1971] [Getting started] [answered]
Note that this is not about whether I can install multiple versions of a package, but of homebrew itself.

I'm on an M1 Mac (Silicon/arm CPU), and as such I'm still having some trouble with packages that break when they're run natively. I'd like to [install Homebrew using Rosetta](https://hackernoon.com/apple-m1-chip-how-to-install-homebrew-using-rosetta-su12331b) (Apple's emulation software), which will then install the relevant packages also under Rosetta.

I am worried that I might end up breaking my current brew installs by doing so. Preferably, I'd make an alias to `rosetta-brew` so that I don't confuse things too much.

Is that possible?

**Comment by nandahkrishna** [comment, marked as answer]
Rosetta and ARM installations of Homebrew can coexist, the former will be installed into `/usr/local` which is the default prefix on Intel and the latter will be installed to `/opt/homebrew`, the default prefix for ARM. You should be able to create a `rosetta-brew` alias which does `arch -x86_64 /usr/local/bin/brew ...` for convenience.

**Reply by thomasaarholt** [reply]
Fantastic. Thanks!
